addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -60,6 +60,9 @@
         return [{
             content: "Open the select",
             trigger: `we-select:has(we-button[${data}]) we-toggler`,
+            run() {
+                this.anchor.click();
+            }
```

Comment:
same here, can't it be `run: "click",`?

Reply:
Otherwise the tour fails. It could be perfect to use it. To be investigated...